tensornetwork/tn_keras/mpo.py

```diff
@@ -127,7 +127,7 @@ def is_perfect_root(n, n_nodes):
         trainable=True,
         initializer=self.bias_initializer) if self.use_bias else None
 
-  def call(self, inputs: tf.Tensor) -> tf.Tensor:
+  def call(self, inputs: tf.Tensor, **kwargs) -> tf.Tensor: # pylint: disable=unused-argument
```

Comment:
Is there a reason for the **kwargs here? The Keras engine should handle not calling this with the wrong argspec.

Reply:
This was added for TF 2.2.0 compatibility as it broke our linter build saying this wasn't there.

Reply:
Hm, that's unfortunate. LGTM, I'll talk to Keras-team about it.

Reply:
That's unfortunate, I'd recommend suppressing the original linter warning for now though, and removing the `**kwargs` here. We'll look into why this changed in 2.2